### PR TITLE
update eukcc treehandler.py script to avoid quoted-node-names issue

### PR DESCRIPTION
**Pull request by chienchi**
closing issue #32 .

### CODE REVIEW

File: Docker/Dockerfile [hunk 1]
```diff
@@ -12,6 +12,8 @@ ENV container docker
 
 ADD * /opt/conda/bin/
 
+CMD mv /opt/conda/bin/treehandler.py /opt/conda/lib/python3.9/site-packages/eukcc/treehandler.py
```
Comment (aclum) [inline Docker/Dockerfile:15]: Where did treehandler.py come from?

Comment (chienchi) [reply Docker/Dockerfile:15]: From the EukCC repo, https://github.com/EBI-Metagenomics/EukCC/blob/master/eukcc/treehandler.py and add quoted node name option to the Tree method.

**Review by aclum — CHANGES_REQUESTED** [review]
Please update to use doejgi/mbin:v1.0.1 as the base image for the Dockerfile. Then we can remove the making these changes on our end including having the treehandler.py file

**Comment by chienchi** [comment]
> Please update to use doejgi/mbin:v1.0.1 as the base image for the Dockerfile. Then we can remove the making these changes on our end including having the treehandler.py file



Yes. Updated Dockerfile, image and the sha256.

**Review by aclum — APPROVED** [review]

**Pull request merged**